File: pset6/mario/more/mario.py
from cs50 import get_int

# Prompt the user to get a height between 1 and 8
while True:
    h = get_int("What's height of the pyramid?" + " ")
    if h > 0 and h < 9:
        break

# Loop through heigth
for i in range(h):
    # For reverse half-pyramid start with creating max space and loop through smaller and smaller spaces
    for j in range(h-i-1):
        print(" ", end="")
    for k in range(i+1):
        print("#", end="")
    print("  ", end="")
    for l in range(i+1):
        print("#", end="")
    print("")

File: pset1/mario/more/mario.c

#include <cs50.h>
#include <stdio.h>


int main(void)
{
    int n;    int l = 1;
    
    do
    {
        n = get_int("Rows: ");
    }
    while (n < 1 || n > 8);                     // get user input between 1 & 8
    

    for (int i = 1; i <= n; i++)                // create n amount of rows
    {
        for (int k = n - i ; k > 0  ; k--)
        {
            printf(" ");
        }
        // output spaces
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int m = 1; m <= i; m++)
        {
            printf("#");
        }
        
        // create bricks
        printf("\n");
    }

}










